shell/common/shell.cc

```diff
@@ -1685,8 +1686,13 @@ bool Shell::OnServiceProtocolGetSkSLs(
     SkBase64::Encode(sksl.second->data(), sksl.second->size(), b64_char);
     b64_char[b64_size] = 0;  // make it null terminated for printing
     rapidjson::Value shader_value(b64_char, response->GetAllocator());
-    rapidjson::Value shader_key(PersistentCache::SkKeyToFilePath(*sksl.first),
-                                response->GetAllocator());
+    std::string_view key_view(reinterpret_cast<const char*>(sksl.first->data()),
```

Comment:
Per the comment I made above about making SKSLCache a struct. I can't tell what sksl.first means here without some context. Just a minor nit.